File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void)
{
    // Height of the pyramid
    int height, row, blank, column;
    do
    {
     height = get_int ("What's the height of the pyramid?\n");
    }
    while (height < 1 || height > 8);

    //first pyramid
for (row = 0; row < height; row++)
{
    for (blank = 0; blank < height - row - 1; blank++)
    {
        printf(" ");
    }
    for (column = 0; column <= row; column++)
    {
        printf("#");
    }

    //second pyramid
    printf("  ");

    for (column = 0; column <= row; column++)
    {
        printf("#");
    }
    //New row
    printf("\n");
}
}

File: sentimental-mario-more/mario.py
# Import get_int
from cs50 import get_int

# Loop to guarantee the correct input
while True:
    height = get_int("What's the height of the pyramid? ")
    if 1 <= height <= 8:
        break

# Print the first pyramid
for row in range(height):
    for blank in range(height - row - 1):
        print(" ", end="")
    for column in range(row + 1):
        print("#", end="")

# Print the second pyramid
    print("  ", end="")
    for column in range(row + 1):
        print("#", end="")
    print("")
